src/services/ui/src/pages/Home.tsx

```diff
@@ -1,3 +1,142 @@
+import {
+  ActionsMenu,
+  Button,
+  Header,
+  Link,
+  Logo,
+  Footer,
+  UsaBanner,
+} from "@enterprise-cmcs/macpro-ux-lib";
```

Comment:
so id recommend bringing everything in as something like:
`import * as UI from "@enterprise-cmcs/macpro-ux-lib"` 

but also it doesnt seem like you are using most of these things in your file. 

it would be nice to use the UsaBanner under the header.